caddyhttp/httpserver/pathcleaner.go

```diff
@@ -0,0 +1,75 @@
+package httpserver
+
+import (
+	"math/rand"
+	"path"
+	"strings"
+	"time"
+)
+
+// A proxy function to prevent one or more of the path cleanup operations:
```

Comment:
What is a "proxy" function? Revise comment to conform to godoc conventions: "CleanMaskedPath prevents ..."